tests/experimental/starry/vis_test.py

```diff
@@ -0,0 +1,16 @@
+import numpy as np
+
+from jaxoplanet.experimental.starry.orbit import SurfaceBody
+from jaxoplanet.experimental.starry.surface import Surface
+from jaxoplanet.experimental.starry.visualization import show_map
+from jaxoplanet.experimental.starry.ylm import Ylm
+
+
+def test_show_map():
+    y = Ylm.from_dense(np.hstack([1.0, np.random.rand(10) * 1e-1]))
+    surface = Surface(y=y, inc=0.9, obl=-0.3, period=1.2, u=[0.5, 0.5])
+    surface_body = SurfaceBody(period=1.0, surface=surface)
+
+    show_map(surface_body)
+    show_map(surface)
+    show_map(y)
```

Comment:
This isn't really related to this PR, but it might be nice to check that this actually shows what we expect it to, rather than just checking that it displays?

Reply:
Any idea of how to do that? Saving an array of the expected result seems overkill right?

Reply:
You could take a look at the tests in corner.py which follows the matplotlib approach: https://github.com/dfm/corner.py/tree/main/tests

But that might be overkill for our case. Your call!